mountaineer/dependencies/__init__.py

```diff
@@ -1,2 +1,7 @@
-from .base import DependenciesBase, get_function_dependencies  # noqa: F401
-from .core import CoreDependencies  # noqa: F401
+from . import core as CoreDependencies  # noqa: F401
```

Comment:
These need to be tagged with `noqa` in addition to using the `import ... as ...` syntax. Ruff will still clear them out during autofixing if they are not commented.